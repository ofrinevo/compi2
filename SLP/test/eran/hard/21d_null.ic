
class MainClass { 

	static void main (string[] args) { 
		
		A a = null;

		Library.println("null.method ");
		Library.printi(a.twice(2));
		Library.println("");
		Library.println("Should not print this or get here.");

	}

}

class A {
 int field;
 int twice(int a) {
  return 2 * a;
 }
}
