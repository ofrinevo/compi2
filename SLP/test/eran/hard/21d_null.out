null.method 
Runtime Error: Null pointer dereference!
